spinalcordtoolbox/moco.py

```diff
@@ -44,7 +44,7 @@ class ParamMoco:
     """
     # The constructor
 
-    def __init__(self, is_diffusion=None, group_size=1, metric='MeanSquares', smooth='1'):
+    def __init__(self, is_diffusion=None, group_size=1, metric='MeanSquares', smooth='1', iter=10):
```

Comment:
We probably want to use the name `iteration` instead, since `iter` is a built-in Python keyword. (I am happy to make this change on your behalf.)